.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "assets/dir-index-html"]
 	path = assets/dir-index-html
-	url = https://github.com/ipfs/dir-index-html.git
+	url = https://github.com/TRON-US/dir-index-html.git
```

Comment:
This doesn't exist right now, where can we set it up? Should it just be a fork of ipfs/dir-index-html to start with?

Reply:
Yep, fork ipfs/dir-index-html and i will submit PR to that

Reply:
\* fork ipfs/dir-index-html and checkout v1.0.5 tag to master

Reply:
@WhiteEgg Updated here: https://github.com/TRON-US/dir-index-html please submit PR there.

Reply:
@taiyangc https://github.com/TRON-US/dir-index-html/pull/2

Reply:
merged, please update reference here

Reply:
@taiyangc update with force push